server/api/ballot_manifest.py

```diff
@@ -116,9 +120,7 @@ def process() -> None:
             CSVColumnType(CVR, CSVValueType.YES_NO, required=use_cvr),
         ]
 
-        # Temporarily wrap file contents in a buffer so we can "stream" it until
-        # we have actual file streaming from storage
-        manifest_file = io.BytesIO(jurisdiction.manifest_file.contents.encode("utf-8"))
+        manifest_file = retrieve_file_contents(jurisdiction.manifest_file)
```

Comment:
as a first-time reader of this code, my instinct is that `retrieve_file_contents` is returning the whole file contents, not a stream. Maybe naming could indicate this more clearly?